packages/components/src/external-link/index.jsx

```diff
@@ -1,88 +1,92 @@
 import { localizeUrl } from '@automattic/i18n-utils';
 import clsx from 'clsx';
 import { translate } from 'i18n-calypso';
```

Comment:
One more little followup: use `useTranslate` instead of the "static" `translate` call. Can be a drive-by change in another `ExternalLink`-related PR.

Reply:
Saw this after merging the other PRs, but here's a follow up: https://github.com/Automattic/wp-calypso/pull/99664